e2e/app-with-domain/app_with_domain_test.go

```diff
@@ -288,6 +290,7 @@ var _ = Describe("App With Domain", func() {
 			var resp *http.Response
 			var fetchErr error
 			urls := strings.Split(route.URL, " or ")
+			expectedResponseBody := "This is Copilot express app for admin"
 			for _, url := range urls {
```

Comment:
this is not from you!!! - but this is not correct if there were multiple urls. all of the `Eventually()` functions will only be looking at the _last_ URL in the list! can you add a line below the for loop like this?
```suggestion
			for _, url := range urls {
			    url := url
```

Reply:
Does running them in parallel make more sense I wonder 🤔 I guess the same logic can be applied to `for _, route := range svc.Routes {`

Reply:
oh good point - it's probably not an issue if `Eventually` doesn't run in parallel...which it probably doesn't?

Reply:
ok it blocks until the condition is true - nvm @paragbhingre ! all good here